WTWSMS/decisions/VIET_roleplay_decisions.txt

```diff
@@ -12,7 +12,6 @@ decisions = {
 		potential = {
 			is_ruler = yes
 			is_landed = yes
-			ai = no
```

Comment:
I found it:
			OR = {
				demesne_size = 2
				higher_tier_than = count
			}

It's already preventing Barons from doing it.